api/Dockerfile.dev

```diff
@@ -0,0 +1,23 @@
+# Use an official Node.js runtime as a parent image
+FROM node:18
```

Comment:
Looks good! Just curious about the node image here and in Dockerfile, what's the difference between using one or the other?

Reply:
The other one uses a lighter version of the image in order to have a reduced image size for the final build.